strings/names/first.txt

```diff
@@ -500,7 +500,6 @@ Garrett
 Gary
 Gavin
 Gaye
-Gaylord
```

Comment:
Considering this is the only edit on this file and it isn't really even bad, it's better to just leave it.